sql/core/src/test/resources/sql-tests/results/postgreSQL/strings.sql.out

```diff
@@ -907,7 +907,7 @@ SELECT rpad('hi', 5, '')
 -- !query 110 schema
 struct<rpad(hi, 5, ):string>
 -- !query 110 output
-hi
+NULL
```

Comment:
Why do you think the current bhaivour is wrong even if PgSQL does so?
```
postgres=# SELECT rpad('hi', 5, '');
 rpad 
------
 hi
(1 row)
```

Reply:
@maropu , yes i know but the correct behaviour as per the definition of  rpad it should return NULL for empty pad.
Mysql : rpad ('hi', 5, '') =>  NULL
Hive : rpad ('hi', 5, '') =>  NULL
Postgresql : rpad ('hi', 5, '')  => hi

Reply:
Ah, I see. Can you check Oracle and presto? They also have NULL for that?

Reply:
please check this link -> https://docs-snaplogic.atlassian.net/wiki/spaces/SD/pages/558530619/Column+Functions+and+Properties+-+Spark+Expression

Reply:
@maropu , wish you a merry christmas

Reply:
ok i will check oracle and presto

Reply:
Oracle : rpad('hi', 5, '') => NULL

Reply:
Thanks for the check. It seems presto throws an exception for that case;
```
presto> select rpad('hi', 5, '');
Query 20191225_040937_00004_htqqc failed: Padding string must not be empty
```

Reply:
Can you describe these behaviours above in the PR description?

Reply:
ok :)